File: decompiled/sources/com/streamax/client/SurfaceCallBackInterface.java
package com.streamax.client;

import android.view.SurfaceHolder;

public interface SurfaceCallBackInterface {
    void surfaceChanged(SurfaceHolder surfaceHolder, int i, int i2, int i3, int i4);

    void surfaceCreated(SurfaceHolder surfaceHolder, int i);

    void surfaceDestroyed(SurfaceHolder surfaceHolder, int i);
}

File: decompiled/sources/com/streamax/client/SurfaceCallBackInterface.kt
package com.streamax.client

import android.view.SurfaceHolder

interface SurfaceCallBackInterface {
    fun surfaceChanged(
        surfaceHolder: SurfaceHolder,
        index: Int,
        format: Int,
        width: Int,
        height: Int,
    )

    fun surfaceCreated(surfaceHolder: SurfaceHolder, index: Int)

    fun surfaceDestroyed(surfaceHolder: SurfaceHolder, index: Int)
}
